spec/system/candidate_fills_in_personal_details_spec.rb

```diff
@@ -0,0 +1,87 @@
+require 'rails_helper'
+
+RSpec.describe 'A candidate filling in their personal details' do
```

Comment:
I'm worried about the readability of this spec. I don't think we should hold up on that account, but I want to have a think about if there's any ways to improve it.

I'm thinking some way to reduce the repeated reference to `application_form.personal_details_section`, but I'm worried about the extra level of abstraction that might introduce.

Reply:
I am ambivalent about this. I like very much that we don't have strings. On the other hand, strings are amazing.

I'd like to hear more MadeTech views on i18n in tests — I've always found it useful but the fact that you don't like it right out of the gate is a (welcome) invitation to reevaluate it.

As far as the code's concerned, if we could dry up the i18n scoping a bit, might that make it a bit better? https://coderwall.com/p/dvme9q/set-scope-of-i18n-translations-in-rails-with-a-block looks interesting

Reply:
I hadn't settled on this being i18n's fault, it's more about the size of the spec. So far, on balance, I like the i18n approach (though I like to battle-test something before I form a strong opinion).

I wonder if we're trying to assert against too much. Would a more lightweight spec (e.g. checking that the page we end up on `has_content` of the name we just entered/the page's header) give us enough confidence this functionality is working?

I asked the Made Tech lot (and linked them here). One suggestion from @craigjbass: consider use of [Page Object](https://martinfowler.com/bliki/PageObject.html) to abstract away some of the details.

Just to re-iterate - I'm definitely happy to merge this and think/improve this as we go!

Reply:
Yes. This is a good thought. Want to ship the other two cards before tomorrow, though. Shall we revisit after that?